### ISSUE TITLE
Crush Form using Edit Frame

### ISSUE THREAD

**Issue by khalil2535** [mypaint/mypaint#1112]
#315 ## Description of the problem

#### Basic system details

MyPaint version: [ v 2.0.0 stable ]
Operating system: [ Arch Linux ]
Desktop environment: [ KDE ]

#### Steps to reproduce

1. Start MyPaint
2. Click on 'Edit Painting Symmetry Icon'
3. Move painting axis
4. Click on Edit Frame

#### Backtraces or error messages

```python
Mypaint version: 2.0.0+gitexport.af5225a
System information: Linux-4.19.146-1-lts419-x86_64-with-glibc2.2.5
Using: Python 3.8.5, GTK 3.24.23, GdkPixbuf 2.40.1, Cairo 1.17.3, GLib 2.66.0
Traceback (most recent call last):
  File "/usr/lib/mypaint/gui/document.py", line 2198, mode_radioaction_changed_cb(self=<gui.document.Document object>, action=<Gtk.RadioAction object at 0x7fa95e8b1780 (GtkRadioAction at 0x55bbd27faeb0)>, current_action=<Gtk.RadioAction object at 0x7fa95e8b1940 (GtkRadioAction at 0x55bbd289ddf0)>)
                    mode = mode_class()
                self.modes.context_push(mode)
  variables: {'self.modes.context_push': ('local', <bound method ModeStack.context_push of <ModeStack [StraightMode, FrameEditMode]>>), 'mode': ('local', <gui.framewindow.FrameEditMode object at 0x7fa95d8b3b50>)}
  File "/usr/lib/mypaint/gui/mode.py", line 1369, context_push(self=<ModeStack [StraightMode, FrameEditMode]>, mode=<gui.framewindow.FrameEditMode object>)
            self._stack.append(mode)
            mode.enter(doc=self._doc)
            self.changed(old=old_mode, new=mode)
  variables: {'mode.enter': ('local', <bound method FrameEditMode.enter of <gui.framewindow.FrameEditMode object at 0x7fa95d8b3b50>>), 'doc': (None, []), 'self._doc': ('local', <gui.document.Document object at 0x7fa95e90f400>)}
  File "/usr/lib/mypaint/gui/framewindow.py", line 116, enter(self=<gui.framewindow.FrameEditMode object>, doc=<gui.document.Document object>, **kwds={})
            """Enter the mode"""
            super(FrameEditMode, self).enter(doc, **kwds)
            # Assign cursors
  variables: {'super': ('builtin', <class 'super'>), 'FrameEditMode': ('global', <class 'gui.framewindow.FrameEditMode'>), 'self': ('local', <gui.framewindow.FrameEditMode object at 0x7fa95d8b3b50>), 'enter': (None, []), 'doc': ('local', <gui.document.Document object at 0x7fa95e90f400>), 'kwds': ('local', {})}
  File "/usr/lib/mypaint/gui/mode.py", line 442, enter(self=<gui.framewindow.FrameEditMode object>, doc=<gui.document.Document object>, **kwds={})
            self.__reset_delta_totals()
            return super(ScrollableModeMixin, self).enter(doc, **kwds)
  variables: {'super': ('builtin', <class 'super'>), 'ScrollableModeMixin': ('global', <class 'gui.mode.ScrollableModeMixin'>), 'self': ('local', <gui.framewindow.FrameEditMode object at 0x7fa95d8b3b50>), 'enter': (None, []), 'doc': ('local', <gui.document.Document object at 0x7fa95e90f400>), 'kwds': ('local', {})}
  File "/usr/lib/mypaint/gui/mode.py", line 1108, enter(self=<gui.framewindow.FrameEditMode object>, doc=<gui.document.Document object>, **kwds={})
                try:
                    self.inactive_cursor = self.doc.app.cursors. \
                            get_action_cursor(self.ACTION_NAME,
  variables: {'self.inactive_cursor': ('local', <gui.framewindow.FrameEditMode object at 0x7fa95d8b3b50>), 'self.doc.app.cursors.get_action_cursor': ('local', <bound method CustomCursorMaker.get_action_cursor of <gui.cursor.CustomCursorMaker object at 0x7fa9630c61f0>>), 'self.ACTION_NAME': ('local', 'FrameEditMode'), 'gui.cursor.Name.ARROW': ('global', 'cursor_arrow')}
  File "/usr/lib/mypaint/gui/cursor.py", line 331, get_action_cursor(self=<gui.cursor.CustomCursorMaker object>, action_name='FrameEditMode', cursor_name='cursor_arrow')
                return Gdk.Cursor.new(Gdk.CursorType.ARROW)
            return self.get_icon_cursor(icon_name, cursor_name)
  variables: {'self.get_icon_cursor': ('local', <bound method CustomCursorMaker.get_icon_cursor of <gui.cursor.CustomCursorMaker object at 0x7fa9630c61f0>>), 'icon_name': ('local', 'mypaint-frame-symbolic'), 'cursor_name': ('local', 'cursor_arrow')}
  File "/usr/lib/mypaint/gui/cursor.py", line 351, get_icon_cursor(self=<gui.cursor.CustomCursorMaker object>, icon_name='mypaint-frame-symbolic', cursor_name='cursor_arrow')
                if "symbolic" in icon_name:
                    icon_pixbuf = gui.drawutils.load_symbolic_icon(
                        icon_name, 18,
  variables: {'icon_pixbuf': (None, []), 'gui.drawutils.load_symbolic_icon': ('global', <function load_symbolic_icon at 0x7fa961d1d940>), 'icon_name': ('local', 'mypaint-frame-symbolic'), 'fg': (None, []), 'outline': (None, [])}
  File "/usr/lib/mypaint/gui/drawutils.py", line 300, load_symbolic_icon(icon_name='mypaint-frame-symbolic', size=16, fg=(1, 1, 1, 1), success=None, warning=None, error=None, outline=(0, 0, 0, 1))
        icon_pixbuf, was_symbolic = info.load_symbolic(
            fg=rgba_or_none(fg),
  variables: {'icon_pixbuf': (None, []), 'was_symbolic': (None, []), 'info.load_symbolic': ('local', gi.FunctionInfo(load_symbolic)), 'fg': ('local', (1, 1, 1, 1)), 'rgba_or_none': ('local', <function load_symbolic_icon.<locals>.rgba_or_none at 0x7fa95e9929d0>), 'success_color': (None, []), 'success': ('local', None), 'warning_color': (None, []), 'warning': ('local', None), 'error_color': (None, []), 'error': ('local', None)}
Error: rsvg-error-quark: XML parse error: error code=201 (3) in (null):26:121: Namespace prefix xlink for href on use is not defined
 (0)
```

**Comment by jplloyd** [comment]
Thanks for reporting, this is a known issue with the 2.0.0 release (this is the seventh duplicate), and was fixed in the last release.

See this comment for further details/references: https://github.com/mypaint/mypaint/issues/1082#issuecomment-633699893

**Comment by randodude123** [comment]
> #315 ## Description of the problem
> 
> #### Basic system details
> MyPaint version: [ v 2.0.0 stable ]
> Operating system: [ Arch Linux ]
> Desktop environment: [ KDE ]
> 
> #### Steps to reproduce
> 1. Start MyPaint
> 2. Click on 'Edit Painting Symmetry Icon'
> 3. Move painting axis
> 4. Click on Edit Frame
> 
> #### Backtraces or error messages
> ```python
> Mypaint version: 2.0.0+gitexport.af5225a
> System information: Linux-4.19.146-1-lts419-x86_64-with-glibc2.2.5
> Using: Python 3.8.5, GTK 3.24.23, GdkPixbuf 2.40.1, Cairo 1.17.3, GLib 2.66.0
> Traceback (most recent call last):
>   File "/usr/lib/mypaint/gui/document.py", line 2198, mode_radioaction_changed_cb(self=<gui.document.Document object>, action=<Gtk.RadioAction object at 0x7fa95e8b1780 (GtkRadioAction at 0x55bbd27faeb0)>, current_action=<Gtk.RadioAction object at 0x7fa95e8b1940 (GtkRadioAction at 0x55bbd289ddf0)>)
>                     mode = mode_class()
>                 self.modes.context_push(mode)
>   variables: {'self.modes.context_push': ('local', <bound method ModeStack.context_push of <ModeStack [StraightMode, FrameEditMode]>>), 'mode': ('local', <gui.framewindow.FrameEditMode object at 0x7fa95d8b3b50>)}
>   File "/usr/lib/mypaint/gui/mode.py", line 1369, context_push(self=<ModeStack [StraightMode, FrameEditMode]>, mode=<gui.framewindow.FrameEditMode object>)
>             self._stack.append(mode)
>             mode.enter(doc=self._doc)
>             self.changed(old=old_mode, new=mode)
>   variables: {'mode.enter': ('local', <bound method FrameEditMode.enter of <gui.framewindow.FrameEditMode object at 0x7fa95d8b3b50>>), 'doc': (None, []), 'self._doc': ('local', <gui.document.Document object at 0x7fa95e90f400>)}
>   File "/usr/lib/mypaint/gui/framewindow.py", line 116, enter(self=<gui.framewindow.FrameEditMode object>, doc=<gui.document.Document object>, **kwds={})
>             """Enter the mode"""
>             super(FrameEditMode, self).enter(doc, **kwds)
>             # Assign cursors
>   variables: {'super': ('builtin', <class 'super'>), 'FrameEditMode': ('global', <class 'gui.framewindow.FrameEditMode'>), 'self': ('local', <gui.framewindow.FrameEditMode object at 0x7fa95d8b3b50>), 'enter': (None, []), 'doc': ('local', <gui.document.Document object at 0x7fa95e90f400>), 'kwds': ('local', {})}
>   File "/usr/lib/mypaint/gui/mode.py", line 442, enter(self=<gui.framewindow.FrameEditMode object>, doc=<gui.document.Document object>, **kwds={})
>             self.__reset_delta_totals()
>             return super(ScrollableModeMixin, self).enter(doc, **kwds)
>   variables: {'super': ('builtin', <class 'super'>), 'ScrollableModeMixin': ('global', <class 'gui.mode.ScrollableModeMixin'>), 'self': ('local', <gui.framewindow.FrameEditMode object at 0x7fa95d8b3b50>), 'enter': (None, []), 'doc': ('local', <gui.document.Document object at 0x7fa95e90f400>), 'kwds': ('local', {})}
>   File "/usr/lib/mypaint/gui/mode.py", line 1108, enter(self=<gui.framewindow.FrameEditMode object>, doc=<gui.document.Document object>, **kwds={})
>                 try:
>                     self.inactive_cursor = self.doc.app.cursors. \
>                             get_action_cursor(self.ACTION_NAME,
>   variables: {'self.inactive_cursor': ('local', <gui.framewindow.FrameEditMode object at 0x7fa95d8b3b50>), 'self.doc.app.cursors.get_action_cursor': ('local', <bound method CustomCursorMaker.get_action_cursor of <gui.cursor.CustomCursorMaker object at 0x7fa9630c61f0>>), 'self.ACTION_NAME': ('local', 'FrameEditMode'), 'gui.cursor.Name.ARROW': ('global', 'cursor_arrow')}
>   File "/usr/lib/mypaint/gui/cursor.py", line 331, get_action_cursor(self=<gui.cursor.CustomCursorMaker object>, action_name='FrameEditMode', cursor_name='cursor_arrow')
>                 return Gdk.Cursor.new(Gdk.CursorType.ARROW)
>             return self.get_icon_cursor(icon_name, cursor_name)
>   variables: {'self.get_icon_cursor': ('local', <bound method CustomCursorMaker.get_icon_cursor of <gui.cursor.CustomCursorMaker object at 0x7fa9630c61f0>>), 'icon_name': ('local', 'mypaint-frame-symbolic'), 'cursor_name': ('local', 'cursor_arrow')}
>   File "/usr/lib/mypaint/gui/cursor.py", line 351, get_icon_cursor(self=<gui.cursor.CustomCursorMaker object>, icon_name='mypaint-frame-symbolic', cursor_name='cursor_arrow')
>                 if "symbolic" in icon_name:
>                     icon_pixbuf = gui.drawutils.load_symbolic_icon(
>                         icon_name, 18,
>   variables: {'icon_pixbuf': (None, []), 'gui.drawutils.load_symbolic_icon': ('global', <function load_symbolic_icon at 0x7fa961d1d940>), 'icon_name': ('local', 'mypaint-frame-symbolic'), 'fg': (None, []), 'outline': (None, [])}
>   File "/usr/lib/mypaint/gui/drawutils.py", line 300, load_symbolic_icon(icon_name='mypaint-frame-symbolic', size=16, fg=(1, 1, 1, 1), success=None, warning=None, error=None, outline=(0, 0, 0, 1))
>         icon_pixbuf, was_symbolic = info.load_symbolic(
>             fg=rgba_or_none(fg),
>   variables: {'icon_pixbuf': (None, []), 'was_symbolic': (None, []), 'info.load_symbolic': ('local', gi.FunctionInfo(load_symbolic)), 'fg': ('local', (1, 1, 1, 1)), 'rgba_or_none': ('local', <function load_symbolic_icon.<locals>.rgba_or_none at 0x7fa95e9929d0>), 'success_color': (None, []), 'success': ('local', None), 'warning_color': (None, []), 'warning': ('local', None), 'error_color': (None, []), 'error': ('local', None)}
> Error: rsvg-error-quark: XML parse error: error code=201 (3) in (null):26:121: Namespace prefix xlink for href on use is not defined
>  (0)
> ```

I'm having the same issue with a similar setup using Gnome at this present time.

**Comment by jplloyd** [comment]
Yes, this was fixed in 2.0.1. You are using 2.0.0. Refer to the link in my comment just above yours.
